pythonforandroid/recipes/freetype/__init__.py

```diff
@@ -35,9 +35,6 @@ def get_recipe_env(self, arch=None, with_harfbuzz=False):
                 'harfbuzz', self.ctx
             ).get_build_dir(arch.arch)
             freetype_install = join(self.get_build_dir(arch.arch), 'install')
-            env['CFLAGS'] = ' '.join(
-                [env['CFLAGS'], '-DFT_CONFIG_OPTION_USE_HARFBUZZ']
-            )
```

Comment:
Removed this macro because it's already defined with the latest version of `freetype`, and,  if we keep it, will throw a warning during the compilation